packages/pieces/community/activepieces/src/index.ts

```diff
@@ -13,10 +17,20 @@ Activepieces Platform API is available under the Platform Edition.
 You can get your API Key from the Platform Dashboard.
 `;
 
-export const activePieceAuth = PieceAuth.SecretText({
-  displayName: 'API Key',
+export const activePieceAuth = PieceAuth.CustomAuth({
   description: markdown,
   required: true,
+  props: {
+    apiKey: PieceAuth.SecretText({
+      displayName: 'API Key',
+      required: true,
+    }),
+    baseApiUrl: Property.ShortText({
+      displayName: 'Base URL',
+      required: true,
+      defaultValue: 'https://cloud.activepieces.com/api/v1',
+    }),
+  },
```

Comment:
I don't have permission to write to your repo, so can you please just place the api key under the base api url ?